imqsauth/http.go

```diff
@@ -702,20 +782,28 @@ func httpHandlerSetPassword(central *ImqsCentral, w http.ResponseWriter, r *http
 }
 
 func httpHandlerResetPasswordStart(central *ImqsCentral, w http.ResponseWriter, r *httpRequest) {
-	identity := strings.TrimSpace(r.http.URL.Query().Get("identity"))
-	code, msg := central.ResetPasswordStart(identity, false)
+	userId, getUserIdErr := getUserId(r)
+	if getUserIdErr != nil {
+		authaus.HttpSendTxt(w, http.StatusBadRequest, "Invalid userid")
```

Comment:
Same thing here.. and in a few other places - always include err.Error(). Some day, somebody trying to use our API will thank you.